modules/ROOT/pages/authentication-authorization/password-and-user-recovery.adoc

```diff
@@ -1,6 +1,7 @@
 :description: This page describes how to reset a password to recover a user's access when their password is lost. It specifically focuses on how to recover an admin user if all the admin users have been unassigned the admin role, and how to recreate the built-in admin role if it has been dropped.
+:page-aliases: configuration/password-and-user-recovery.adoc
 [[password-and-user-recovery]]
-= Password and user recovery
+=  Recover user and password
```

Comment:
```suggestion
=  Recover admin user and password
```

Reply:
Ah yes, but the links are still functioning because of the alias. In any case, it's good to have this updated in the page itself. Thanks for flagging it!